package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "build-all": "yarn run build-ui && yarn run build-css",
-    "postinstall": "npm i && cd css && npm i",
+    "postinstall": "",
```

Comment:
The infinite loop was caused by the first `npm i && `. `postinstall` only runs after `npm i` is called so running it from a `postinstall` will result in an infinite loop.

With a `postinstall` of `"cd css && npm i"`, you basically end up with:
```bash
alias npm i="npm i && cd css && npm i"
```

With a `postinstall` of just `"npm i"`, you'd end up with something like:
```bash
alias npm i="npm i && npm i && npm i && npm i && npm && ..."
```

I'm not sure if my example is clear, but